### PR TITLE
php-cs-fixer 2.16.1

### PR DESCRIPTION
**Pull request by derrabus**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Thanks @derrabus

File: Formula/php-cs-fixer.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class PhpCsFixer < Formula
   desc "Tool to automatically fix PHP coding standards issues"
   homepage "https://cs.sensiolabs.org/"
-  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.0/php-cs-fixer.phar"
-  version "2.16.0"
-  sha256 "59a5f7fe8c6e946fdd7426c39e11d502c48549b1c79e87ddebdbf9d4c022c194"
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.1/php-cs-fixer.phar"
+  version "2.16.1"
```
Comment (glensc) [inline Formula/php-cs-fixer.rb]: isn't the `version` superfluous?

works fine without:
- https://github.com/Homebrew/homebrew-core/pull/46550/files

Comment (Bo98) [reply Formula/php-cs-fixer.rb]: Looks like Homebrew's version detection function detects `cs` as the version from this particular URL:

```ruby
> Version.detect("https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.1/php-cs-fixer.phar", {})
=> #<Version::FromURL:0x00007fc08209cc58 @version="cs">
```

Comment (derrabus) [reply Formula/php-cs-fixer.rb]: Homebrew can guess the version from the __file name__, but not from the path. The file name does not contain the version, thus we need to specify it.

**Pull request merged**